resources/js/Components/navigation/Footer.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+    <div>
+        <footer class="bg-white">
+            <div class="mx-auto w-full max-w-screen-xl p-8">
+                <ACMCertLogo class="sm:hidden"></ACMCertLogo>
+                <ul class="grid grid-cols-2 gap-8 py-6 sm:py-0 sm:flex sm:justify-between">
+                    <ul>
```

Comment:
<ul> semantic is define list
```suggestion
                <div class="grid grid-cols-2 gap-8 py-6 sm:py-0 sm:flex sm:justify-between">
                    <ul>
```